doc/groups.md

```diff
@@ -1 +1,519 @@
 # Groups Module
+Module | HTTP Method | URL | Description 
+--- | --- | --- | ---
+[Add Groups](#add) | POST | /groups | Add Groups API
+[Edit Groups](#edit) | PUT | /groups/{id} | Edit Groups API
+[View All Groups](#view) | GET | /groups | View Groups API
+[Delete Groups](#delete) | DELETE | /groups/{id} | Delete Groups API
+
+
+### Database
+![](./groups-layer.png)
```

Comment:
- remove aro_id in groups table
- remove kunci in aros table

Reply:
okay sir, i remove them